packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card.tsx

```diff
@@ -39,6 +46,7 @@ import { useBlockProtocolCreatePropertyType } from "../../../../../../components
 import { useBlockProtocolUpdatePropertyType } from "../../../../../../components/hooks/block-protocol-functions/ontology/use-block-protocol-update-property-type";
 import { StyledPlusCircleIcon } from "../../../../shared/styled-plus-circle-icon";
 import { useRouteNamespace } from "../../../../shared/use-route-namespace";
+import { useEntityType } from "../shared/entity-type-context";
 import { EntityTypeEditorForm } from "../shared/form-types";
 import {
   usePropertyTypes,
```

Comment:
Can we rename this to `useLatestPropertyTypes`, if that's accurate? I think it would be easier to tell what's going on when we have both `entityTypesAndPropertyTypes` and `propertyTypes`

Reply:
Renamed in 3d5787116febb8c5d3726720beb99487ac471e1d